cdm-test/src/test/java/ucar/nc2/dt/grid/TestGridSubset.java

```diff
@@ -202,50 +199,10 @@ public void testMSG() throws Exception {
     }
   }
 
-  @Test
-  @Ignore("Bad URL, as of 2015/03/11.")
```

Comment:
:-/

Reply:
Are you reacting to the bad URL, or that the test was removed?

FYI, the test is not removed; merely moved to `TestGridSubsetThredds`.

Reply:
The bad url comment :-)